pyproject.toml

```diff
@@ -23,8 +23,8 @@ classifiers = [
 
 dependencies = [
     "packaging",
-    "numpy>=1.22.4,<2.0.0",
```

Comment:
The dependency versions need to be fixed ("numpy>=1.22.4", "quantities>=0.16.1").

Reply:
it was more the lower limit I was talking about, which was going back to 1.19 when I reviewed this.